Codec/Compression/BZip/Internal.hs

```diff
@@ -284,7 +401,11 @@ compressStream (CompressParams blockSize workFactor initChunkSize) =
 
       Stream.Error _ msg -> fail msg
 
-
+-- | The unfolding of the decompression process, where you provide a sequence
+-- of compressed data chunks as input and receive a sequence of uncompressed
+-- data chunks as output. The process is incremental, in that the demand for
+-- input and provision of output are interleaved.
+--
```

Comment:
Since this was not exported previously, could you please add `@since@ annotation?